dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.authData-2.0.feature

```diff
@@ -0,0 +1,12 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
```

Comment:
Can you make this file and symbolic name be io.openliberty instead of com.ibm.websphere.appserver?

Reply:
I did originally name this one and the 1.0 with the io.openliberty prefix, but the build complained that some APIs were missing from the removed 1.0 feature. I talked to @tjwatson, and he said I could get around it by going to WS-CD-Open and changing some stuff over there and coming back and delivering this after, but to be honest, that seemed like a pain to simply rename a private feature and I didn't want the two version of the feature to have different long names.

Reply:
furthermore, you cannot have the two features use different names because then they would not collide with each other as singletons.

Reply:
Technically they do not need to be singletons, but am ok if this doesn't end up happening.